MUXSDKStats/MUXSDKStatsTests/MUXSDKStatsTests.m

```diff
@@ -843,19 +850,19 @@ - (void) testOrientationAndRenditionChangeEventSequence {
 
     [self assertPlayer:playName dispatchedDataEvents:@{
         @(1): [NSNull null],
-        @(4): @{BANDWIDTH: @(258157)},
-        @(6): @{BANDWIDTH: @(1927853)},
-        @(9): @{BANDWIDTH: @(258157)},
+        @(5): @{BANDWIDTH: @(258157)},
```

Comment:
how come we are still sending the same renditions even though now we are waiting for playback to start in order to send a renditionchange?

Reply:
I think essentially this is because of how the test works, the test forces the rendition change notifications so the change is always registered cause the playbackStartDate logic happens before this. So not on testing but on an actual flow if there is a rendition change that we get in the accessLog we check for playbackStartDate and then depending on that value we post a notification with the rendition change or just return earlier without posting the notification. However, these tests are not mocking the accessLog but instead directly forcing the rendition change notifications.

Reply:
Just wanted to confirm here. We're forcing the `RenditionChangeNotification` with valid data so as Steph said the tests are working with that notification instead of the accessLog, with the access log there's more logic to check to see if we can send or not this rendition notification but in this case the tests will just sent the notification with the data we want to test and verify.